.github/workflows/deploy-export-template.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
   deploy-export-template:
```

Comment:
Did you check everything is working when deploying on a fork repo ?

Reply:
To be honest i didn't. Did not think of the ability to test releases on a fork. Will do that asap (same for the other PR)

Reply:
You were right to ask for testing on a fork. There was an issue with the zip naming. It's now fixed. See: https://github.com/chippmann/godot-kotlin-jvm/releases/tag/untagged-ac089a3da05694148e49

Reply:
Updated url: https://github.com/chippmann/godot-kotlin-jvm/releases/tag/0.5.3-3.5.1-SNAPSHOT